amd_openvx/openvx/ago/ago_kernel_api.cpp

```diff
@@ -2301,9 +2301,6 @@ int ovxKernel_LaplacianPyramid(AgoNode * node, AgoKernelCommand cmd)
     else if (cmd == ago_kernel_cmd_query_target_support) {
         node->target_support_flags = AGO_KERNEL_FLAG_SUBGRAPH
                     | AGO_KERNEL_FLAG_DEVICE_CPU
-#if ENABLE_OPENCL
```

Comment:
why is this removed? Are we supporting Laplacian Pyramid only on CPU?

Reply:
@rrawther As of now yes, that's the whole reason for the test fail